+sig/Signal.m

```diff
@@ -349,7 +345,7 @@
     
   end
   
-  %% Overridden MATLAB Methods
+  %% Overloaded MATLAB Methods
```

Comment:
technically these are overridden, not overloaded methods. "overloaded" describes multiple methods with the same name that can be called from the same object, with the language interpreter using a certain technique to figure out which method should be called (e.g. if the methods take different number of input args or arg types). the methods here are "overridden" because, e.g., there are not two different "sin" methods a signals object can call; instead, the signals object "overrides" MATLAB's built-in "sin" function. 
Do you want to fix this (and the header) back to the way I had it? I'm fine with using "Signals" instead of *signals* : )

Reply:
Or I am happy to make these quick changes and merge? Just want to make sure this makes sense and is fine with you?